examples/funloc/analysis_fun.py

```diff
@@ -26,173 +24,32 @@
 
     * http://lester.ilabs.uw.edu/files/AKCLEE_110_slim.tar.gz
 
-"""
+Setting up a config file
+------------------------
+Parameters for remotely connecting to SSS workstation ('sws') can be set
+by adding a file ~/.mnefun/mnefun.json with contents like:
+
+    $ mkdir ~/.mnefun
+    $ echo '{"sws_ssh":"kasga","sws_dir":"/data06/larsoner/sss_work","sws_port":22}' > ~/.mnefun/mnefun.json
```

Comment:
spaces after colons & commas?

Reply:
also wondering why this is kept in a JSON file separate from the YAML params...  is it because the YAML is meant to be published with the code, whereas this should not?

Reply:
The analysis YAML is for a given project, how to process the data. It should be shareable by each person in the project, and its values should be specific to the given project.

The `mnefun.json` is a for a given machine/workstation, how to do SSS. It shouldn't be shareable across people, but its values should work for any project.